book/styles.md

```diff
@@ -646,8 +647,8 @@ can also hide bugs in your code, so if something's not right try
 removing it temporarily.
 
 ::: {.further}
-Each browser has its own browser style sheet ([Chromium][blink-css],
-[Safari][webkit-css], [Firefox][gecko-css]). [Reset
+Each browser engine has its own browser style sheet ([Chromium][blink-css],
+[WebKit][webkit-css], [Gecko][gecko-css]). [Reset
```

Comment:
Yes, pointing to the open source core instead of Safari is a good thing.

Though... are Chromium / WebKit / Gecko the same type of thing? Or is it Blink / WebKit / Gecko? Like, per my understanding Chromium includes the chrome, but WebKit and Gecko don't. Does WebKit formally include JSC, or is it that Safari combines WebKit and JSC? Same question for Gecko... Basically we would ideally have the three names in this sentence have the same type.

Reply:
Yes those are the same type of thing. Chromium is the official name of the code repo.